src/Foundation/Connect/tests/Models/Mappings/SearchProfileTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿namespace Wooli.Foundation.Connect.Tests.Models.Mappings
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using AutoMapper;
+
+    using Ploeh.AutoFixture;
+
+    using Sitecore.Commerce.Engine.Connect.Search;
+    using Sitecore.Commerce.Engine.Connect.Search.Models;
+    using Sitecore.ContentSearch.Linq;
+    using Sitecore.ContentSearch.SearchTypes;
+
+    using Wooli.Foundation.Connect.Models.Mappings;
+    using Wooli.Foundation.Connect.Models.Search;
+    using Xunit;
+
+    using FacetValue = Connect.Models.Search.FacetValue;
+
+    public class SearchProfileTests
+    {
+        private readonly IMapper mapper;
+
+        private readonly IFixture fixture;
+
+        public SearchProfileTests()
+        {
+            var configuration = new MapperConfiguration(
+                cfg => { cfg.AddProfile<SearchProfile>(); });
+
+            this.mapper = new Mapper(configuration);
+
+            this.fixture = new Fixture();
+        }
+
+        [Fact]
+        public void Configuration_ShouldBeValid()
+        {
+            // arrange
+            var configuration = new MapperConfiguration(
+                cfg => { cfg.AddProfile<SearchProfile>(); });
+
+            // act, assert
+            configuration.AssertConfigurationIsValid();
+        }
+
+        [Fact]
+        public void Map_IfCommerceQueryFacetObjectIsValid_ShouldReturnValidQueryFacetObject()
+        {
+            // arrange
+            var commerceQueryFacet = new CommerceQueryFacet()
```

Comment:
Will `this.fixture.Create<CommerceQueryFacet>()` work here?

Reply:
Yes, I'll replace this